src/Umbraco.Web.UI.Client/src/views/common/infiniteeditors/mediapicker/mediapicker.html

```diff
@@ -145,11 +144,57 @@
                     </umb-empty-state>
 
                 </div>
+            </form>
 
+        </umb-editor-container>
+        <!--
+        <umb-editor-container ng-if="vm.activeTab.alias === 'clipboard'">
+
+            <div style="margin-bottom:20px; text-align: right;">
+                <umb-button type="button"
+                            icon="icon-trash"
+                            button-style="link"
+                            label-key="clipboard_labelForClearClipboard"
+                            action="vm.clickClearClipboard()">
+                </umb-button>
+            </div>
```

Comment:
We should remove this unused code

Reply:
Funky, interesting that Patrick and I wrote almost identical code, resulting in merging resulted in this.